api/catalog/api/serializers/media_serializers.py

```diff
@@ -277,7 +277,7 @@ class TagSerializer(serializers.Serializer):
         help_text="The name of a detailed tag.",
     )
     accuracy = serializers.FloatField(
-        required=False,
+        default=None,
         help_text="The accuracy of a machine-generated tag. Human-generated "
```

Comment:
A bit of a nit-pick: @dhruvkb Would we need to update this documentation to say "Human-generated tags have a `null` accuracy field" rather than not having the field at all?

Reply:
Hey @sarayourfriend, thank you for the review will update the code once we finalise the help text.

Reply:
+1 to that, I've added a suggestion for that wording.

Reply:
Hey @AetherUnbound, @sarayourfriend  and @dhruvkb, I have made the required changes, and the request has also passed all the mandatory tests. Thank you for your help and suggestions.

Reply:
Excellent, thank you @sahil-R!